slave/buildslave/commands/base.py

```diff
@@ -147,7 +155,7 @@ def __init__(self, builder, stepId, args):
         self.args = args
         self.startTime = None
 
-        missingArgs = filter(lambda arg: arg not in args, self.requiredArgs)
```

Comment:
PEP8 doesn't forbid lambdas!  It forbids the assignment of a lambda directly to a local variable - the fifth bullet point in https://www.python.org/dev/peps/pep-0008/#programming-recommendations

So actually, neither of the lambdas you're addressing here violate that recommendation -- in `test_fiximports.py` the lambda is assigned to an instance attribute (not a bare identifier), while in `base.py` the lambda is not assigned to anything.